README.rst

```diff
@@ -144,14 +137,14 @@ is produced, something is wrong.
 By default, tests are prevented from overusing resources like disk space and
 memory.  To enable these tests, run ``make testall``.
 
-IMPORTANT: If the tests fail and you decide to mail a bug report, *don't*
+IMPORTANT: If the tests fail and you decide to file a bug report, *don't*
 include the output of ``make test``.  It is useless.  Run the failing test
 manually, as follows::
```

Comment:
@matrixise I'm not sure I understand your point.  I switched to `make test TESTOPTS="-v test_that_failed"` because running the tests is suggested with `make test`, and there was previously a note about what to do if you built python in a different directory.  Reusing `make test` with `TESTOPTS` means the different directory question just goes away, but we get exactly the same result.